tests/specs/graph/JsrSpecifiers_FastCheck_UnsupportedDestructuring.txt

```diff
@@ -0,0 +1,65 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+const other = { a: Math.random(), b: Math.random() };
+interface Other {
+  a: number;
+  b: number;
+}
+export const { a, b }: Other = other;
```

Comment:
This is for simplicitly in fast check at the moment and in the future it could be relaxed. The suggestion is to break this up to not destructure because otherwise things start to get really complicated to analyze.